config/modules/cloudonly.conf

```diff
@@ -0,0 +1,15 @@
+# Cloud only tests
+
+# Default built-in tests.
+include ${DAQ_LIB}/config/modules/host.conf
```

Comment:
If this is really intended to be cloud-only, then this shouldn't include the base host tests and remove others.  What happens if something else is added into host.conf?  Then the logic would end up not being "cloud-only"...

Reply:
I've changed this to only build core modules and cloud test module